client/src/featureform/serving.py

```diff
@@ -816,6 +816,38 @@ def from_stub(self, name, version, model: Union[str, Model] = None):
         stream = Stream(self._stream, name, version, model)
         return Dataset(stream)
 
+    def dataframe(self) -> pd.DataFrame:
```

Comment:
should we get rid of the `pandas` method?

Reply:
or better yet. have pandas call this method

Reply:
bump ^ was this resolved?